_data/changelogs/docs-migration-v3.yml

```diff
@@ -2,6 +2,11 @@ title: Migrating to 3.0
 type: documentation
 changelogURL:
 items:
+  - date: 2023-11-29
```

Comment:
> [!IMPORTANT]
> Neds updated changelog date on approval

Reply:
Resolved in [a41ec1a](https://github.com/uswds/uswds-site/pull/2319/commits/a41ec1a53e6079f994cf24ef9a89dc5febc87253)